.gitignore

```diff
@@ -1,4 +1,6 @@
 /vendor
+/contrib/vendor
+/envoy/vendor
```

Comment:
Not sure that we need this, or "/vendor", since there's no local vendoring in they repo AFAICT.

Reply:
Ah yea I can delete that thanks for the catch. Was just testing stuff